.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ executors:
         default: "11.0"
     working_directory: ~/code
     docker:
-      - image: cimg/openjdk:<<parameters.jdk>>
+      - image: alvrme/alpine-android:android-31-jdk11-v2021.07.14
```

Comment:
Not easily parametrized...

Reply:
we need to figure out how to parameterize this since we want to test for build-time compatibility with java 8

Reply:
can we use the orb? https://circleci.com/developer/orbs/orb/circleci/android

alternatively, it's not very hard to install android sdk from the cli

Reply:
Turns out there are alternate equivalent images that we use without dates in them, so we can parameterize over them.